scripts/ngrok.ts

```
@@ -0,0 +1,41 @@
+import * as ngrok from "@ngrok/ngrok";
+import { config } from "dotenv";
+import { expand } from "dotenv-expand";
+import { z } from "zod/v4";
```

ESLint

[error] Unable to resolve path to module 'zod/v4'.

(import-x/no-unresolved)